specifications/api/swagger.yaml

```diff
@@ -2533,6 +2574,22 @@ paths:
         - *parametersIntendedStakeAmount
       responses: *responsesListStakePools
 
+    post:
```

Comment:
What about nesting this a bit more? A `POST` on `/stake-pools` suggests the creation of a stake pools. But it's not. What about having this under: `/stake-pools/maintenance-actions` instead ?

Reply:
Shall we than have `GET /stake-pools/maintenance-actions` to return the last_gc time there? Atm this was added into `GET /stake-pools`

Reply:
Why was this not discussed in the ticket? https://jira.iohk.io/browse/ADP-478

I clearly described my design decisions there and asked for review :)

Reply:
Some of it was discussed. And the notation is ambiguous enough to leave room for interpretation (as in, saying "I'll add an endpoint to `/wallets` can be read as "under the namespace `/wallets`" and not "at this exact path"). Next time, provide an OpenAPI excerpt to remove all ambiguity :+1:

Reply:
> rationale: this re-uses an existing endpoint and potentially allows to add more actions without the need of a new endpoint

Wasn't really ambiguous I think